docs/guides/transpiler-stages.ipynb

```diff
@@ -32,11 +32,10 @@
     "qiskit-addon-sqd~=0.8.1\n",
```

Comment:
<p>Might be worth setting seeds in this notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2584/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>